src/components/RouteButton.js

```diff
@@ -49,13 +64,13 @@ const RouteButton = props => {
         <View style={styles.body}>
           <Tts text={props.title} id={`${props.title}-tts`} dontShowText />
           <View style={styles.button}>
-            <Button icon={<Icon type='font-awesome-5' name={props.icon} size={25} color={Colors.primary} />} title={props.title} titleStyle={styles.buttonTitle} buttonStyle={{ borderRadius: 15, paddingTop: 10 }} type='outline' onPress={props.handleScreen} />
+            <Button icon={<Icon type='font-awesome-5' name={props.icon} size={25} color={Colors.primary} />} title={props.title} titleStyle={styles.buttonTitle} buttonStyle={{ borderRadius: 15, paddingTop: 10 }} type='outline' onPress={navigationHandler()} />
```

Comment:
Do not call function immediately- either use `onPress={navigationHandler}` or `onPress={() => navigationHandler()}`